baseplate/secrets/fetcher.py

```diff
@@ -267,8 +270,13 @@ def main():
             os.fchmod(f.fileno(), cfg.output.mode)
 
             json.dump({
-                "vault_token": client.token,
                 "secrets": secrets,
+                "vault": {
+                    "token": client.token,
+                    "url": cfg.vault.url,
+                },
+                # this is here for backwards compatibility
+                "vault_token": client.token,
```

Comment:
What does leaving this here actually do?  The implementation doesn't seem to check this if ['vault']['token'] is not present.